openid-4-verifiable-presentations-1_0.md

```diff
@@ -2257,7 +2312,9 @@ The `vp_formats` parameter of the Verifier metadata MUST have the key `jwt_vp_js
 
 This section illustrates the presentation of a Credential conformant to [@VC_DATA] that is signed using JWS, and does not use JSON-LD.
 
-The Credential format identifiers are `jwt_vc_json` to request a W3C Verifiable Credential and `jwt_vp_json` to request a W3C Verifiable Presentation.
+The Credential Format Identifiers are `jwt_vc_json` to request a W3C Verifiable Credential and `jwt_vp_json` to request a Verifiable Presentation compliant to the [@VC_DATA] specification.
+
+If `jwt_vc_json` is used, the Credential is returned without Holder Binding. Therefore, the Verifier MUST set the `allow_replay` parameter in the Credential Query to `true` if it is using `jwt_vc_json`. Otherwise, `allow_replay` MUST be `false`. Wallets MUST reject requests that do not follow these rules.
```

Comment:
I feel like `allow_replay` is more of a result of not requiring key binding. But I don't think it's the best name for a query parameter.

I'm not sure if there's an overarching term. I think in OID4VCI it's called key binding (so something like 'require_key_binding`) or more generic that a presentation is required (so `require_verifiable_presentation`).

Reply:
I don't think `require_key_binding` sounds serious enough :-) I'd prefer a wording that conveys that this is a critial parameter.

Reply:
I agree with @TimoGlastra .